content/en/agent/kubernetes/daemonset_setup.md

```diff
@@ -188,7 +193,58 @@ To enable [Log collection][10] with your DaemonSet:
     (...)
     ```
 
-2. Mount the `pointdir` volume in *volumeMounts*:
+2. Mount the Docker socket or `/var/log/pods`
+
+The Agent has two ways to collect logs: from the Docker socket, and from the Kubernetes log files (automatically handled by Kubernetes). 
+
+Use log file collection when:
+
+* Docker is not the runtime
+* More than 10 containers are used within each pod 
+
+The Docker API is optimized to get logs from one container at a time. When there are many containers in the same pod, collecting logs through the Docker socket might be consuming much more resources than going through the files.
+
+{{< tabs >}}
+{{% tab "K8s File" %}}
+
+    ```
+      (...)
+        volumeMounts:
+          (...)
+          - name: logpath
+              mountPath: /var/log/pods
+      (...)
+      volumes:
+        (...)
+        - hostPath:
+            path: /var/log/pods
+            name: logpath
+      (...)
+    ```
+    
+{{% /tab %}}
+{{% tab "Docker Socket" %}}
+
+    ```
+      (...)
+        volumeMounts:
+          (...)
+          - name: dockersocket
+              mountPath: /var/run/docker.sock
+      (...)
+      volumes:
+        (...)
+        - hostPath:
+            path: /var/run/docker.sock
+            name: dockersocket
+      (...)
+    ```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+
+3. Mount the `pointdir` volume in *volumeMounts*:
```

Comment:
Is this supposed to be `pointdir` or `pointerdir`? It's `pointerdir` in the snippet below.

Reply:
Thanks for spotting this, let's use the same name than in the snipper (the actual name does not matter but it must be the same everywhere).